api/app/Console/Commands/SuspendPlacedCandidates.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enums\PoolCandidateStatus;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class SuspendPlacedCandidates extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:suspend-placed-candidates';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Set suspended at date to now() for placed term and indeterminate candidates';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dateNow = Carbon::now();
+        $applicableStatuses = [PoolCandidateStatus::PLACED_TERM->name, PoolCandidateStatus::PLACED_INDETERMINATE->name];
+
+        DB::beginTransaction();
+        try {
+            DB::table('pool_candidates')
+                ->whereIn('pool_candidate_status', $applicableStatuses)
+                ->whereNull('suspended_at')
+                ->update(['suspended_at' => $dateNow]);
```

Comment:
This will be less accurate if the command takes longer than one second to complete.  I'd recommend using a database expression like `CURRENT_TIMESTAMP` instead.

Reply:
Still applicable with the eloquent approach?

Reply:
Ah, well then not sure how you would add it in then?

Reply:
If you're grabbing Carbon::now() inside of the loop it's probably fine since it will stay current.  The other option that I was originally recommending was something like we do for UUIDs:
```php
new Expression('public.gen_random_uuid()')
```

Reply:
Stick with Carbon in that case?

Reply:
It's fine I think.